src/DriverBase.cpp

```diff
@@ -215,6 +263,15 @@ int DriverBase::setupAnalogTPDOs(std::vector<canbus::Message>& messages,
     return pdoIndex;
 }
 
+int DriverBase::setupEncoderTPDOs(std::vector<canbus::Message>& messages,
+    int pdoIndex, canopen_master::PDOCommunicationParameters const& parameters) {
+    pdoIndex = setupEncoderTPDOsInternal(
+        m_expected_encoder_counter_mask, 0, messages,
+        pdoIndex, parameters
+    );
```

Comment:
The creation of an Internal method here does not bring anything. It was present for Analog because of the duplication between converted and raw analog readings.